build.cake

```diff
@@ -1295,6 +1297,148 @@ Task("bindings-verify")
+		        "CustomView",
+		        "DataBinding",
+		        "DataBindingAdapters",
+		        "DataBindingCommon",
+		        "DataBindingRuntime",
+		        "ViewBinding",
+		        "DocumentFile",
+		        "DrawerLayout",
+		        "DynamicAnimation",
+		        "Emoji",
+		        "ExifInterface",
+		        "GridLayout",
+		        "HeifWriter",
+		        "Interpolator",
+		        "Leanback",
+		        "V14",
+		        "UI",
+		        "Utils",
+		        "V13",
+		        "V4",
+		        "LiveData",
+		        "ViewModel",
+		        "ViewModelSavedState",
+		        "LocalBroadcastManager",
+		        "Media2",
+		        "MediaRouter",
+		        "MultiDex",
+		        "Runtime",
+		        "PercentLayout",
+		        "RecyclerView",
+		        "SavedState",
+		        "SlidingPaneLayout",
+		        "Sqlite",
+		        "SwipeRefreshLayout",
+		        "TvProvider",
+		        "VectorDrawable",
+		        "VersionedParcelable",
+		        "ViewPager",
+		        "ViewPager2",
+		        "WebKit",
+            };
+            var dictionary_custom = WeCantSpell.Hunspell.WordList.CreateFromWords(words);
+
+            using (StreamReader reader = System.IO.File.OpenText(@"./config.json"))
+            {
+                JsonTextReader jtr = new JsonTextReader(reader);
+                binderator_json_array = (JArray)JToken.ReadFrom(jtr);
+            }
+
+            List<string> spell_errors = new List<string>();
+
+            Information("config.json spell checking...");
+
+            foreach(JObject jo in binderator_json_array[0]["artifacts"])
+            {
+                bool? dependency_only = (bool?) jo["dependencyOnly"];
+                if ( dependency_only == true)
+                {
+                    continue;
+                }
+                string nuget_id  	= (string) jo["nugetId"];
+                Information($"       spell-check {nuget_id}");
+
+                string[] nuget_id_parts = nuget_id.Split('.');
+                foreach(string nuget_id_part in nuget_id_parts)
+                {
+                    bool check_dictionary_custom = dictionary_custom.Check(nuget_id_part);
+                    Information($"      check_dictionary_custom {nuget_id_part} = {check_dictionary_custom}");
+                    if (check_dictionary_custom)
+                    {
+                        Information($"          Found in custom dictionary!");
+                        continue;
+                    }
+                    bool check_dictionary = dictionary.Check(nuget_id_part);
+                    Information($"      check_dictionary {nuget_id_part} = {check_dictionary}");
+                    if (check_dictionary)
+                    {
+                        Information($"          Found in EN-US dictionary!");
+                        continue;
+                    }
+                    spell_errors.Add(nuget_id_part);
+                    // var suggestions = dictionary.Suggest("teh");
```

Comment:
Looks like some leftover debugging code.

Reply:
That was more API exploration code from the demo sample...
Removed.

Reply:
Removed